generatebundlefile/hack/release.sh

```diff
@@ -45,7 +45,7 @@ ${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
     --input ${BASE_DIRECTORY}/generatebundlefile/data/input_121.yaml \
     --release-profile ${PROFILE}
 
-# Create the prod bundle
+# Create the prod bundle for 1.21
 ${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
```

Comment:
`{BASE_DIRECTORY}/generatebundlefile/generatebundlefile` instead? I don't see bin folder generated on my side.

Reply:
The Make build creates a bin folder